packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -1374,6 +1518,132 @@ class SyncWatcher extends BaseWatcher {
   private _isL2BridgeContract (bridgeContract: L1BridgeContract | L2BridgeContract): bridgeContract is L2BridgeContract {
     return !this.isL1
   }
+
+  private async getRelayerFee (destinationChain: Chain): Promise<BigNumber> {
+    // TODO: Introduce this post-nitro
+    // if (destinationChain === Chain.Arbitrum) {
+    //   return this.getArbitrumRelayGasCost(destinationChain)
+    // }
+
+    return BigNumber.from(0)
+  }
+
+  private async _getRelayGasCost (): Promise<BigNumber> {
+    if (this.chainSlug === Chain.Arbitrum) {
+      return this._getArbitrumRelayGasCost()
+    }
+
+    throw new Error('Unsupported relay chain')
+  }
+
+  private async _getArbitrumRelayGasCost (): Promise<BigNumber> {
```

Comment:
suggestion: I think this function (and associated functions) should be moved to a separate util since this is very arbitrum specific and it can be unit tested separately

Reply:
Resolved in 6f6cc07f0a419b7c2d18accbd3969aa1e951e472